### DISCUSSION TITLE
Please share any barriers you may have experienced and/or suggestions for how we could improve the accessibility of Instructor Training Checkout?

### DISCUSSION THREAD

**Discussion by klbarnes20** [carpentries/instructor-training#1439] [checkout-feedback]
### Background
When someone wants to be a certified Carpentries Instructor, they first take Instructor Training. After that, they must complete checkout to be certified.

**[Checkout](https://carpentries.github.io/instructor-training/checkout/) consists of three steps:**

- Attend a community discussion

- Make a contribution to a Carpentries lesson (including those in the incubator) or other resources

- Do a 5 minute teaching demo 

The Core Team is working on a project that is the first systematic review and opportunity to improve the checkout process since 2018. **We want to hear from you!**

**Comment by sarasrking** [comment]
Timezone issues can be a barrier, if community discussions fill up into the distance from when someone is due to checkout could mean that they just don't end up participating as they forget or get busy. And GitHub is definitely a barrier if you've never seen it before doing the Carpentries training. A GitHub intro session from time to time could be helpful for those wanting to get through that first hurdle. People also get super nervous doing the 5 min demo so any chance to practice between the workshop and the checkout might be advantageous (though I would usually recommend my trainees buddy up with some people from their cohort to do that themselves, also as a way of building community locally).

**Reply by fiveop** [reply]
I want to pick up on the idea of GitHub intro sessions.

For the two instructor trainings I co-taught, we skipped the website creation exercise (which mostly involved working with GitHub), but offered extra sessions in the week after the workshop for those who wanted help with the exercise or had any questions regarding GitHub. The reasoning behind it was that there usually is a split between those people who already now GitHub or similar tools, for which the exercise and the checkout requirement is close to trivial, and those that need to get acquainted with a new tool first, which is a much bigger task.

I suggest that something like these sessions becomes an official part of the instructor training related events, just like the teaching demonstrations; however, in this case, optional.
